impress-org/givewp
# Refactor: update strings translations (@DAnn2012)

Resolves [GIVE-1159]

## Description



This includes all PR submissions from @DAnn2012 to update string translations
- #7496
- #7497
- #7498 
- #7499
- #7500
- #7501
- #7502
- #7513


## Affects



Strings in following locations:
- Donation and Subscription list tables
- Form builder embed
- Form summary block
- Form builder email options
- GiveWP global advanced settings
- Recurring donations promo in form builder

## Visuals



N/A

## Testing Instructions



- Run through automated tests
- Make sure affected pages work as expected

## Pre-review Checklist






[GIVE-1159]: https://stellarwp.atlassian.net/browse/GIVE-1159?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
